docs/multicluster/user-guide.md

```diff
@@ -442,21 +436,12 @@ ConfigMap `antrea-mc-controller-config-***`. In other member clusters, a
 ClusterInfoImport CR will be created for the cluster which includes the
 exported network information. For example, in cluster `test-cluster-west`, you
 you can see a ClusterInfoImport CR with name `test-cluster-east-clusterinfo`
-is created for cluster `test-cluster-east`. You can check the
-`ClusterInfoImport` with command: `kubectl get clusterinfoimport -o yaml`,
-which should show information like:
+is created for cluster `test-cluster-east`. You can check it with command:
```

Comment:
Maybe remove "You can check it with command".

`%s/. You can check it with command:/:`

Reply:
Done